src/commands/image/imageSource/ContainerAppUpdateStep.ts

```diff
@@ -87,4 +88,8 @@ export class ContainerAppUpdateStep<T extends ImageSourceContext & IngressContex
     public shouldExecute(context: T): boolean {
         return !!context.containerApp;
     }
+
+    public addExecuteSteps(): AzureWizardExecuteStep<T>[] {
```

Comment:
Is there any reason to not just add this step upfront when we create the wizard?